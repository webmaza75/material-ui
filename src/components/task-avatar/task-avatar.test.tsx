import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import TaskAvatar from './task-avatar';

describe(`TaskAvatar`, () => {
  it(`TaskAvatar renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const props = 'Payload Monitoring Report';
    const result = renderer
      .render(<TaskAvatar props={props} />);

    expect(result).toMatchSnapshot();
  });
});
