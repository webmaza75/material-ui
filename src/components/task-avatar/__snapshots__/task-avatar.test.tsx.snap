// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskAvatar TaskAvatar renders correctly 1`] = `
<WithStyles(ForwardRef(Avatar))
  className="makeStyles-avatar-1 makeStyles-avatarNA-5"
>
  NA
</WithStyles(ForwardRef(Avatar))>
`;
